README.md

```diff
@@ -30,9 +30,10 @@ Docker compose follow steps: [docker compose](https://docs.docker.com/compose/in
 
 #### 2.3 Install ruby in user app
 
-##### 2.3.1 Change user using
+##### 2.3.1 Change user using and add user to docker group
 ```bash
 su - app
+sudo usermod -aG docker $USER
```

Comment:
I think it's better to add this line to the previous section(Docker installation)
Also, the `app` user is not supposed to have sudo rights due to security concerns

Reply:
added to the docker part without `sudo`